### PR TITLE
[TRACKER-286] Adding surefire config to increase permgen space

### PR DESCRIPTION
**Pull request by abhinavbansal246**

### CODE REVIEW

**Comment by abhinavbansal246** [comment]
http://builds.cask.co/browse/TRAC-BT32-1

File: pom.xml [hunk 1]
```diff
@@ -269,6 +269,27 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <version>${maven-surefire.version}</version>
+        <configuration>
+          <argLine>-Xmx5000m -Djava.awt.headless=true -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -XX:OnOutOfMemoryError="kill -9 %p" -XX:+HeapDumpOnOutOfMemoryError</argLine>
+          <redirectTestOutputToFile>${surefire.redirectTestOutputToFile}</redirectTestOutputToFile>
```
Comment (poornachandra) [inline pom.xml]: This is not defined, is it false by default

Comment (abhinavbansal246) [inline pom.xml]: yes

**Comment by poornachandra** [comment]
LGTM

**Pull request merged**